examples/platforms/simulation/radio.c

```diff
@@ -221,6 +221,42 @@ static uint16_t crc16_citt(uint16_t aFcs, uint8_t aByte)
     return (aFcs >> 8) ^ sFcsTable[(aFcs ^ aByte) & 0xff];
 }
 
+
+#if OPENTHREAD_SIMULATION_VIRTUAL_TIME == 1
+void reportRadioStatusToOTNS(void)
```

Comment:
One suggestion related to naming. to use `reportRadioStatusToOtns()` instead of `ToOTNS`

In terms of naming style, we generally treat abbreviations as full name and use camel-case for them. Many examples in OT code: `Dns`, `Srp`, `Dhcp6`, `Slaac`, etc.

Reply:
Thank you for this point @abtink. I couldn't agree more with this suggestion.

Reply:
I guess this is already resolved